docs/examples/embed.js

```diff
@@ -0,0 +1,65 @@
+/*
+	An example of how you can send embeds
+*/
+
+// Extract the required objects from the discord.js module
+const { Client, MessageEmbed } = require('discord.js');
+
+// Create an instance of a Discord client
+const client = new Client();
+
+// The token of your bot - https://discordapp.com/developers/applications/me
+const token = 'your bot token here';
```

Comment:
since you dont really load it from somewhere but hardcode it, you could just put it in client.login() directly